stdlib/bytes.mli

```diff
@@ -173,6 +173,28 @@ val mapi : (int -> char -> char) -> bytes -> bytes
     index (in increasing index order) and stores the resulting bytes
     in a new sequence that is returned as the result. *)
 
+val fold_left : ('a -> char -> 'a) -> 'a -> bytes -> 'a
+(** [fold_left f x s] computes
+    [f (... (f (f x s.[0]) s.[1]) ...) s.[n-1]],
```

Comment:
Strictly speaking this comment is wrong because the syntax `s.[i]` does not work with `bytes`.